person_capture/gui_app.py

```
@@ -4592,32 +4619,58 @@ def save_hit(c, idx):
                         f"crop@{idx}: face_box={c.get('face_box')}", key="cropface", interval=2.0
                     )
 
+                    processed_crop_xyxy = (int(cx1), int(cy1), int(cx2), int(cy2))
+                    source_size = self._capture_source_size(cap, frame.shape[:2])
```

Ruff (0.15.11)

[warning] Function definition does not bind loop variable `frame`

(B023)